tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -5065,11 +5079,14 @@ <h2>Show similarity to selected datapoint</h2>
         if (!this.local) {
           const url = tf_backend.addParams(
             '/data/plugin/whatif/infer_mutants', urlParams);
-          const chartMakerCallback = function(result){
+          const chartMakerCallback = result =>
```

Comment:
nit: for legibility, if an arrow function is more than one line long, please use braces.
i.e., 
```js
// A bit hard to read
foo = result => 
    iAmGoingToReturnSomething(
       result);
// To below
foo = result => {
    return iAmGoingToReturnSomething(
       result);
};
```

Here and L5086

Reply:
done